src/ui/settings/components/InstallPluginAlert.tsx

```diff
@@ -0,0 +1,64 @@
+import { clipboard } from "@metro/common";
+import { installPlugin } from "@lib/plugins";
+import { findByProps } from "@metro/filters";
+import { Forms } from "@ui/components";
+
+const { FormInput } = Forms;
+const Alert = findByProps("alertDarkStyles", "alertLightStyles").default;
+
+const Alerts = findByProps("openLazy", "close");
+
+export function showInstallPluginAlert() {
+	Alerts.openLazy({
+		importer: async function () {
+			return () => <InstallPluginAlert />;
+		}
+	});
+};
+
+const HTTP_REGEX = /^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&\/=]*)$/;
```

Comment:
Perhaps move this into our `lib/constants.ts`? It may be useful for plugins in future, and will reduce copied code.